activators/lex/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+import plugins.publish.POM_DESCRIPTION
+import plugins.publish.POM_NAME
+
+ext[POM_NAME] = "JAICF-Kotlin Lex Activator Adapter"
+ext[POM_DESCRIPTION] = "JAICF-Kotlin Lex Activator Adapter. Provides intent recognition and named entity extraction."
+
+plugins {
+    `jaicf-kotlin`
+    `jaicf-publish`
+    `jaicf-junit`
+}
+
+dependencies {
+    core()
+    api(jackson())
+    api(platform("software.amazon.awssdk:bom:2.14.3"))
+    api("software.amazon.awssdk:lexruntimev2:2.15.69")
+
+    testImplementation("io.mockk:mockk" version { mockk })
+    testApi("org.junit.jupiter:junit-jupiter-api" version { jUnit })
```

Comment:
There is no need to declare testApi/testRuntime for junit. They are already provided by `jaicf-junit` plugin.